rationale.md

```diff
@@ -37,25 +55,26 @@ Purely for illustration purposes, not intended to represent actual releases:
 
 - `opentelemetry-api` 1.0.0
   - Contains APIs for tracing, baggage, propagators, context
-- `opentelemetry-tracing` 1.0.0
-  - Contains the tracing SDK
 - `opentelemetry-sdk` 1.0.0
   - Contains SDK components for tracing, baggage, propagators, and context
 
 ##### Contains the following experimental packages
 
-- `opentelemetry-api-metrics` 0.x.y
-  - Contains the EXPERIMENTAL API for metrics. There are no stability guarantees.
-- `opentelemetry-metrics` 0.x.y
-  - Contains the EXPERIMENTAL SDK for metrics. There are no stability guarantees.
+- `opentelemetry-api` 0.x.y-b0
```

Comment:
My understanding here was that we were going to version this 1.15b0, but i don't have a strong preference for make this version greater than the released version. I guess the only confusing part here is that this experimental package would include a stable 1.0.0 + the metrics signal

Reply:
I think that may be the more preferable path. This method does enable the stable packages to release alpha/beta/rc versions, if it is necessary.